src/main/java/seedu/address/model/schedule/Event.java

```diff
@@ -84,7 +99,8 @@ public boolean equals(Object other) {
 
     @Override
     public String toString() {
-        return String.format("%s %s %s %s", eventDescription, date, time, duration);
+        return String.format("%s %s %s-%s", eventDescription,
```

Comment:
Is it better if the full day event or events that ends the next day handled separately?

Reply:
Yup you are right, I was thinking of how to best display this as well.

Reply:
Perhaps if it is on the next day, you could add (+1) at the end to indicate it is the next day? Cos we are avoiding displaying the date agiain in the end right

Reply:
Okay sounds good, I will do that.